_config.yml

```diff
@@ -1 +1 @@
-exclude: [src]
+exclude: ["src","highlight","hooks"]
```

Comment:
"highlight" should not be excluded here (or we need to update the HTML files to remove references to it)